labstats/bin/init.py

```diff
@@ -55,12 +55,15 @@
     SELECT `user`, `host`, `start` FROM `session` WHERE `end` IS NULL;
 
 CREATE VIEW `users_in_lab_count_public` AS
-    SELECT count(*) as `count` FROM `users_in_lab`;
+    SELECT COUNT(DISTINCT `user`) AS `count` FROM `users_in_lab`;
 
 CREATE VIEW `staff_in_lab_public` AS
     SELECT `user`, `host`, `start` FROM `users_in_lab` WHERE `user` IN (
         SELECT `user` FROM `staff`
     );
+    
+CREATE VIEW `staff_in_lab_count_public` AS
+    SELECT COUNT(DISTINCT `user`) AS `count` FROM `staff_in_lab_public`; 
```

Comment:
is this a useful view? this can already be calculated from public views pretty easily. normally we've only been using views when we want to provide data that can only be calculated from the private data, or is hard to calculate

if you want to keep this, probably need to add a grant below